src/Stipple.jl

```diff
@@ -608,15 +615,16 @@ const max_retry_times = 10
 
 """
     Base.push!(app::M, vals::Pair{Symbol,T}; channel::String,
-                except::Union{Genie.WebChannels.HTTP.WebSockets.WebSocket,Nothing,UInt}) where {T,M<:ReactiveModel}
+                except::Union{Nothing,UInt,Vector{UInt}}) where {T,M<:ReactiveModel}
 
 Pushes data payloads over to the frontend by broadcasting the `vals` through the `channel`.
 """
 function Base.push!(app::M, vals::Pair{Symbol,T};
-                    channel::String = Genie.config.webchannels_default_route,
-                    except::Union{Genie.WebChannels.HTTP.WebSockets.WebSocket,Nothing,UInt} = nothing)::Bool where {T,M<:ReactiveModel}
+                    channel::String = getchannel(model),
```

Comment:
Is `model` here correct? Where is it coming from?

Reply:
no, it should be `app`

Reply:
We had `Genie.config.webchannels_default_route` before and that generated errors, whenever I used it. I don't think that we still need the default route here, it's from the early days. Do you agree?
If yes, I'll fix that to be `app`